tests/llmcompressor/modeling/test_fuse.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import torch
+
+from llmcompressor.modeling.fuse import center_embeddings, fuse_norm_linears
+
+
+@pytest.mark.unit
+def test_center_embeddings():
+    embedding = torch.nn.Embedding(10, 10)
+    center_embeddings(embedding)
+
+    assert torch.allclose(
+        embedding.weight.mean(dim=1), torch.zeros(embedding.num_embeddings), atol=1e-5
+    )
+
+
+@pytest.mark.unit
+def test_fuse_norm_linears():
+    norm = torch.nn.LayerNorm((5,))
```

Comment:
`torch.nn.LayerNorm` defaults weights to identity unless updated:
```
In [2]: a=torch.nn.LayerNorm((5,))

In [3]: a
Out[3]: LayerNorm((5,), eps=1e-05, elementwise_affine=True)

In [4]: a.weight
Out[4]: 
Parameter containing:
tensor([1., 1., 1., 1., 1.], requires_grad=True)
```
So i think this would be a better test
```suggestion
    norm = torch.nn.LayerNorm((5,))
    norm.weight.data = torch.rand((5,))
```